documentation/smg/02_development_environment.md

```diff
@@ -1 +1,102 @@
+* You need to build Horace and Herbert. With both cloned as in the instructions above, open a Powershell and cd to the main Horace repo directory. Then type
+  * `.\tools\build_config\build.ps1 -build -vs_version 2019`
+* The build will tailor the `horace_on` and `herbert_on` commands to fit the location of the current repo.
+* You can now access Horace: cd to Horace\build\local_init and type `horace_on`. To avoid this indirect access, you can set this directory to be searched by setting the environment variable MATLABPATH to the full directory path before launching MATLAB. `horace_on` can then be used immediately on launch.
+
+### IDaaS environment
+
+#### Access
+
+* Location for login and registration is https://isis.analysis.stfc. This should open on the login page - as a new user you may get more info going directly to the registration, but if you don't, the FAQ menu at the top will take you to a page with "How do I get a user office account" three quarters of the way down the left "Contents" sidebar. This section has a link to a page with sign-in and registration areas. Bottom right click on "Create a new facility user account with us".
+* On this registration page, fill in the form. Email should be your STFC email. For "Facilities You Use" select ISIS. Click "Create New Account" at the bottom. You will probably get an email from FacilitiesBusinessSystem@stfc.ac.uk asking you to validate your account - please do.
+* After this you should be able to log in and create a password. Login is your STFC email address. Password will NOT be your federated password. You already have the federated password for access with your federated user id (form: aaa11111), and a UKSBS Oracle password for access with your  Oracle user id (form aaaa11@ph.rc). Choose whatever password you think is appropriate for this additional login which is not the same as the other two. NB the federated password never seems to change but is obscure; you have to change the Oracle password every 90 days.
+
+#### Opening a VM
+
+* You will see a page "Analysis Environments" hopefully open at the second tab "Analysis". You will see at least one choice "Excitations Single Crystal". Pull down `Launch` bottom right and select "Open in Browser". You should get a new window with a Linux desktop.
+* In the VM desktop, the Applications menu is bottom left on a taskbar, followed by icons for your open windows (at the moment you have none). Click Applications/System to get a terminal. Click Software/Matlab to get a Matlab IDE window. Matlab will (at time of writing) be R2018b.
+* Note that the Horace item in Applications/Software is the GUI version.
+* Note that Matlab will be launched with the system Horace pre-loaded. To move to a development version, type `horace_off`,    change directories to the development version and type `horace_init` to get Horace loaded from your git repos.
```

Comment:
```suggestion
* Note that Matlab will be launched with the system Horace pre-loaded. To move to a development version, type `horace_off`, change directories to the development version and type `horace_init` to get Horace loaded from your git repos.
```

Reply:
You'll also need to call `herbert_off`, `cd` into your version of Herbert and call `herbert_init` before calling `horace_init`. Otherwise, you'll still be using the system version of Herbert.

Reply:
Well noted.